octopoes/octopoes/xtdb/client.py

```diff
@@ -96,6 +97,40 @@ def get_entity(self, entity_id: str, valid_time: Optional[datetime] = None) -> d
         self._verify_response(res)
         return res.json()
 
+    def get_entity_history(
+        self,
+        entity_id: str,
+        *,
+        sort_order: str = "asc",  # Or: "desc"
+        with_docs: bool = False,
+        has_doc: Optional[bool] = None,
+        offset: int = 0,
+        limit: Optional[int] = None,
+        indices: Optional[List[int]] = None,
+    ) -> List[TransactionRecord]:
+        params = {
+            "eid": entity_id,
+            "sort-order": sort_order,
+            "history": "true",
+            "with-docs": "true" if with_docs else "false",
+        }
+
+        res = self._session.get(f"{self.client_url()}/entity", params=params)
+        self._verify_response(res)
+
+        transactions: List[TransactionRecord] = parse_obj_as(List[TransactionRecord], res.json())
+
+        if has_doc is True:  # The doc is None if and only if the hash is  "0000000000000000000000000000000000000000"
+            transactions = [transaction for transaction in transactions if transaction.contentHash != 40 * "0"]
+
+        if has_doc is False:  # The doc is None if and only if the hash is  "0000000000000000000000000000000000000000"
+            transactions = [transaction for transaction in transactions if transaction.contentHash == 40 * "0"]
```

Comment:
The same statement is being executed in both cases `has_doc` is not `None`

Reply:
Which statement?

Reply:
Line 124 and 127 are duplicates

Reply:
No the first is a `!=` and the second `==`